### PR TITLE
feat(@cubejs-backend/mysql-aurora-serverless-driver): Add a new driver to support AWS Aurora Serverless MySql

### PR DESCRIPTION
**Pull request by kcwinner**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

* feat: Adds a MySql driver for Amazon Aurora Serverless DB cluster

### CODE REVIEW

File: packages/cubejs-server-core/core/CompilerApi.js [hunk 1]
```diff
@@ -58,7 +58,10 @@ class CompilerApi {
     options = options || {};
     const { includeDebugInfo } = options;
     const dbType = this.getDbType();
+
+    console.log('DB TYPE:', dbType);
```
Comment (ovr) [inline packages/cubejs-server-core/core/CompilerApi.js:62]: Can you please remove this debug line? Thanks

File: packages/cubejs-schema-compiler/adapter/QueryBuilder.js [hunk 2]
```diff
@@ -34,6 +35,7 @@ const ADAPTERS = {
   elasticsearch
 };
 exports.query = (compilers, dbType, queryOptions) => {
+  console.log('BUILDING QUERY');
```
Comment (ovr) [inline packages/cubejs-schema-compiler/adapter/QueryBuilder.js:38]: Can you please remove this debug line? Thanks

**Review by ovr — CHANGES_REQUESTED** [review]

File: packages/cubejs-server-core/core/CompilerApi.js (hunk 1 above)
Comment (kcwinner) [reply packages/cubejs-server-core/core/CompilerApi.js]: Whoops! Removed them

**Comment by paveltiunov** [comment]
@kcwinner Hey Kenneth! Thanks for contributing this! I'd suggest to call it `mysql-aurora-driver` explicitly.

**Comment by kcwinner** [comment]
Renamed to `mysql-aurora-serverless`. Tests run. Tested locally with `yarn link` to a new test project connected to my db

**Comment by paveltiunov** [comment]
@kcwinner Looks great! Thanks!

**Comment by keydunov** [comment]
Hey @kcwinner, thank you for contributing this!

We may want to identify it as a community-supported driver and it would be great if you'd be able to continue maintaining it. @rpaik will share with you more details about it and our upcoming contributors program.

Thank you again for contributing, it looks really good!

**Comment by kcwinner** [comment]
@keydunov @rpaik I also joined the slack. I figured that will be easier to communicate

**Pull request merged**